dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -874,8 +874,7 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     sign_in @student
 
     get :show, script_id: sl.script, stage_id: sl.stage, id: sl, solution: true
-
-    assert_response :forbidden
+    assert_select '.teacher-panel', 0
```

Comment:
Hmm I'm not sure I see how this is better.  Only teachers should have that link, and the only time it would be `403 Forbidden` would be if their login expired.  In that case we should redirect to the login page.

It seems like we're optimizing for the less-common use case: a student with a teacher link, vs a logged-out teacher.